.gitignore

```diff
@@ -23,3 +23,4 @@ packaging/rpm/shopify-cli.spec
 ext/shopify-extensions/shopify-extensions
 ext/javy/bin
 .console_history
+TODO.md
```

Comment:
I added this one because I like to have a `TODO.md` at the root with notes for myself with the next steps.